src/gpu/generic/sycl/README.md

```diff
@@ -93,6 +93,14 @@ The implementation supports both forward and backward directions.
 `pow`, `relu`, `round`, `soft_relu`, `sqrt`, `square`,`swish` and `tanh`
 * Supported formats: `NCDHW`, `NDHWC`, `NCHW`, `NHWC`, `NCW`, `NWC`, `NC`, `N`
 * Supported data types: `f32`, `bf16`, `f16`, `s32`, `s8`, `u8`
+  
+## Inner Product
+
+The implementation supports the forward direction only.
+
+* Supported formats: All plain formats are supported.
+* Supported data types: All possible data combinations as listed in the specification are supported.
+* Supported post-ops: All the post-ops as mentioned in the specification are supported.
```

Comment:
```suggestion
* Supported post-ops: All the post operations as mentioned in the specification are supported.
```

Reply:
I have a slight preference towards the phrase `post-ops`, as the oneDNN specification too uses `post-ops` rather than post operations

Reply:
> I have a slight preference towards the phrase `post-ops`, as the oneDNN specification too uses `post-ops` rather than post operations

The general writing guidelines advise spelling out and expand the contracted terms. If this is a specialized term, suggest including it in code blocks for example, `post-ops` or italicize the term _post-ops_. In addition, we can call out that we are referring to the oneDNN specification for context. 

Supported _post-ops_: All the _post-ops_ mentioned in the oneDNN specification are supported.